File: templates/matlab/radar_advanced_stage_step.m
function fig = radar_advanced_stage_step()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('step_curve', 4117, 'advanced radar analysis: stage step curve', 'advanced radar analysis', 'stage step curve');
end

File: templates/python/radar_advanced_stage_step.py
"""radar_advanced_stage_step: 雷达进阶阶段阶梯曲线（step 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='advanced radar analysis: stage step curve'):
    return make_template_figure('step_curve', seed=4117, title=title, domain='advanced radar analysis', topic='stage step curve')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
